### ISSUE TITLE
Bug with skinwalkers mod

### ISSUE THREAD

**Issue by TheRealMrKam** [BananaPuncher714/LethalQuantities#33]
It seems there is a bug between Lethal Quantities and Skinwalkers, it's a shame since they are both god tier mods. When i use both the skinwalker mod doesn't seem to care about the config file and puts either everything on true or everything on false depending the version (i tried with multiple versions of both mods)

**Comment by BananaPuncher714** [comment]
Can you elaborate? Do you mean that every enemy has voices on? I haven't tested it myself, but this seems rather serious.

**Comment by TheRealMrKam** [comment]
Yes depending of the version on the mods they have either all voices on or all voices off and i tested with only those 2 mods. 
On Lethal Quantities i'm using scrap and daytime, enemies and outside spawns modifications
On skinwalkers i'm using only the masked on true end frequency 2 
I need more testing tho, i'll do that tomorrow !

**Comment by BananaPuncher714** [comment]
Ok thanks! I'm not entirely sure why they would all have voices on or off, but when I get the time I'll try to debug it myself, with the latest version of skinwalkers.

As for the scrap and daytime enemy configs, the rarity is currently bugged, so if you don't put in a value for the rarity, it will default to 0. It _should_ be the same as issue #29 , although if your case is different, please open another ticket!

**Comment by TheRealMrKam** [comment]
Thanks !

I looked into issue 29 but i think it's a different issue because i didn't eneble global config

**Comment by BananaPuncher714** [comment]
I think I should be able to fix the skinwalkers incompatibility next update, for the latest version of both mods. It seems like when I cloned certain enemies, it changed the name of the GameObject, which skinwalkers uses to check if the enemy is enabled. As a result, skinwalkers used the other enemies option for all enemies.

For the daytime and scrap configs, can you provide a profile code or your configs? What values are you trying to change? The daytime enemies is currently bugged unless you set the rarity for every enemy, but the scrap config should work without error, assuming both are enabled in the global Configuration.cfg file.

**Comment by TheRealMrKam** [comment]
Oh i see, nice ! Thanks again :)

For the other issue it was enabled on configuration.cfg file but i think i understand why it wasn't working but i didn't had time to try it out yet.
I ported my configs from 1.0.14 and in the new version the rarity setting changed enplacement before it was like this : 

[EnemyTypes.HoarderBug]
Rarity = 70

and i still have this one but it changed and now like this :

## Rarity of a(n) Hoarding bug relative to the total rarity of all other enemy types combined. A higher rarity increases the chance that the enemy will spawn. The default value is 0. This option can inherit from the global config with the value GLOBAL and from the game with the value DEFAULT. If no value or DEFAULT is present in the global config, then this value will be the DEFAULT for this moon.
# Setting type: String
# Default value: GLOBAL
HoarderBug = 

Same thing for scraps.
So i assume i now have to put the value there, I just didn't see i had 2 rarity settings...

For Enemies.cfg it was working because i only changed the global SpawnAmountCurve there and that didn't change emplacement.

Maybe i should delete the files and re-generate a clean version of it.

**Comment by BananaPuncher714** [comment]
Yes, ideally you would back up your old files and port over everything to the new files. I moved the rarity section to its own section since I imagined it would be more convenient to have them all in the same location.

**Comment by TheRealMrKam** [comment]
Yeah i think it's definitely a good idea for ergonomy, maybe even better if you put the sentence only once to explain the rarity section and then you have all the settings. It's also important to have the vanilla value to compare with it to know what you want more of etc... kinda like this :


[Rarity]

## Rarity of an enemy relative to the total rarity of all other enemy types combined. A higher rarity increases the chance that the enemy will spawn. The default value is 0. This option can inherit from the global config with the value GLOBAL and from the game with the value DEFAULT. If no value or DEFAULT is present in the global config, then this value will be the DEFAULT for this moon.
# Setting type: String

#BaboonHawk Default value: 15
BaboonHawk = 

#Blob Default value: 20
Blob = 

#Centipede Default value: 60
Centipede = 


etc...

**Comment by BananaPuncher714** [comment]
For sections like rarity, that's a good idea. I'll implement it next update, and provide the sentence on the first enemy(sorted alphabetically). I did notice that reading the sentence over and over again seemed rather annoying. I hope that people don't accidentally skip it though.

**Comment by TheRealMrKam** [comment]
Haha yeah it's easy to skip things and there is a lot of things in those cfgs
